### ISSUE TITLE
Copy nullable state of tuple elements independently

### ISSUE THREAD

**Issue by cston** [dotnet/roslyn#32600]
Warnings should not be reported for `t.Item2.F` or `v.Item2.F`:
```C#
struct S
{
    internal object? F;
}

class Program
{
    static void F()
    {
        (object, S) t = (new S(), new S() { F = 2 });
        t.Item2.F.ToString(); // ok

        (S, S) u = (new S(), new S() { F = 2 });
        (object, S) v = u;
        v.Item2.F.ToString(); // ok
    }
}
```

**Comment by jaredpar** [comment]
Related to #30995

**Comment by jcouv** [comment]
This seems to have been fixed already ([sharplab](https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQBMQGoA+BiAdgVwBtCJhC4ACOXU8gWACgABAJgoEFGBvRivipgGYBARgBsFAJZgADoUkBjSTAoB7GYggxVCCgCEA/AAp2FCAEoKAXgB8FAsQDcjAL6NW+7q8awE+BSoAyl4M/FK48Ag0hGrAAFZwAQYUAGLODG7MbEwiAOwhYTkSTAAsqUbmvPw8oWH8RnpIFIEGlipWFEa4cADuHBVN3X2BFRRcqdYUIhQu5o4UAPQLFD0QUZK4AOYgZlBQkpu4G5scFNqetXVnAHQAkvBgItcAKqqBMAjHFfNXfEsrayOWx29zgjwokAAnhRgJQHIQqlcYHcHixrgA1CCEfBwa4pF5vD5fOaLZaqADWiJmriAA===)). @cston can you confirm and close?

**Comment by jcouv** [comment]
This seems to have been fixed already. @cston can you confirm and close?
